claim/README.md

```diff
@@ -194,6 +194,14 @@ might be having with the ACLK or claiming process.
 
 Use these keys and the information below to troubleshoot the ACLK.
 
+#### bash: netdata-claim.sh: command not found
+
+If you run the claiming script and see a `command not found` error, you either installed Netdata in a non-standard
+location or are using an unsupported package. If you installed Netdata in a non-standard path using the `--install`
+option, you need to update your `$PATH`, or `cd` to the path and run `netdata-claim.sh` there. If you are using an
```

Comment:
Strictly speaking, the user just has to use the absolute path to the command. For example, if they installed to `/opt/netdata` (as the static install does), they would need to run `/opt/netdata/bin/netdata-claim.sh` instead of `netdata-claim.sh`.

Reply:
Changed it. Thanks!